src/Uno.UI/UI/Xaml/Internal/InputManager.cs

```diff
@@ -5,16 +5,37 @@
 #nullable enable
 
 using System;
+using System.Collections.Generic;
+using System.Numerics;
+using Microsoft.UI.Composition;
+using Microsoft.UI.Composition.Interactions;
+using Microsoft.UI.Input;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Input;
+using Microsoft.UI.Xaml.Media;
 using Uno.UI.Xaml.Input;
 using Windows.Devices.Input;
+using Windows.Foundation;
 using Windows.UI.Input.Preview.Injection;
-using Microsoft.UI.Xaml;
-using Microsoft.UI.Xaml.Input;
+
+#if !HAS_UNO_WINUI
+using Windows.UI.Input;
+#endif
 
 namespace Uno.UI.Xaml.Core;
 
 internal partial class InputManager : IInputInjectorTarget
 {
+	private sealed class GestureRecognizerOwner(List<InteractionTracker> trackers, InputManager inputManager)
```

Comment:
All of this should be move into the nested `PointerManager` class (and probably in a dedicated file like `InoutManager.Pointers.Composition.cs`). The goal was to avoid to end with a to complex class and isolate each responsibilities into dedicated classes (keyboard, pointers, ...).

Reply:
I'll refactor

Reply:
Done